jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -0,0 +1,141 @@
+                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/redhat/staging-cds/"
+                    ],
+                    [
+                        name: 'MAIL_LIST_SUCCESS',
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'release-engineering@redhat.com',
+                        ].join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+        ]
+    )
+
+    commonlib.checkMock()
+    buildlib.initialize()
+    buildlib.cleanWorkdir("${env.WORKSPACE}")
+
+    try {
+        sshagent(['openshift-bot']) {
+            advisory = params.ADVISORY ? Integer.parseInt(params.ADVISORY.toString()) : 0
+            def components = params.COMPONENTS.replaceAll(',', ' ').split(' ')
+            def filename = "tarball-srouce-output.txt"
+            def outdir = params.OUT_DIR.toString()
+            def rcm_guest = params.RCM_GUEST.toString()
+            currentBuild.description += "tarball: ${components}"
+
+            stage("elliott tarball-source") {
+                component_args = ""
+                components.each {
+                    component_args += "--component ${it}"
+                }
+                buildlib.elliott """
+                    tarball-sources create
+                    --out-dir=${outdir}
+                    --force
+                    ${component_args}
+                    ${advisory} > ${filename}
+                """
+            }
+
+            stage("rsync") {
+                cmd = "rsync -avz --chmod=go+rX ${outdir} ${rcm_guest}"
+                commonlib.shell(
+                    script: cmd,
+                    returnAll: true
+                )
+            }
+
+            stage("file ticket to RCM") {
+                cmd = """awk '/${advisory}/ { save=\$0 }END{ print save }' ${filename}"""
+                verify = commonlib.shell(
+                        script: cmd,
+                        returnStdout: true
+                )
+
+                description = """
+                    Hi RCM,
+                    The OpenShift ART team needs to provide sources for `${components}` in https://errata.devel.redhat.com/advisory/${advisory}
+                    This release is urgent and we want it to be unblocked and shipped ASAP.
+
+                    The following sources are uploaded to rcm-guest.app.eng.bos.redhat.com:/mnt/redhat/staging-cds/
```

Comment:
hardcoded. if we're gonna have the param we should at least use it here :) but i don't think we really even need it to be a param.

Reply:
this one is a little different from `outdir`